### PR TITLE
Show state of actions on cient.

### PR DESCRIPTION
**Pull request by Lunarmagpie**
This PR adds more actions that the client can see the result of before receiving conformation from the server. I plan to do a few more actions and clean this up a bit, it should be ready for 8.0.

This is temporary until I am able to implement dead reckoning for all actions.

### CODE REVIEW

**Comment by Benji42** [comment]
Is this supposed to be a draft?

**Review by Screaper91 — CHANGES_REQUESTED** [review]

File: client/src/logic/game/local-state.ts [hunk 1]
```diff
@@ -0,0 +1,148 @@
...
+
+		if (!board.activeRow) {
+			board.activeRow = board.rows[row].entity
+		}
+
+		// If we placed a hermit and its not the first turn, we can probably do an attack so lets make the
+		// button visible
+		if (gameState.turn.turnNumber > 1) {
+			gameState.turn.availableActions.push('PRIMARY_ATTACK', 'SECONDARY_ATTACK')
+		}
```
[the remark this answers is not in the document]
Comment (Lunarmagpie) [reply client/src/logic/game/local-state.ts:44]: Good catch. Thank you!

**Comment by Lunarmagpie** [comment]
Im actually going to go up and update card energies so they are not mangled on the frontend. This will make this system know if you are able to play an attack effectively.

**Comment by Lunarmagpie** [comment]
This works well besides for pick requests, which we can't show the result of without large reworks. Doing this properly will take a lot of work, probably a few weeks.

**Pull request merged**